src/components/Nav.js

```diff
@@ -5,24 +5,33 @@ import {
   Route,
   Link,
 } from 'react-router-dom';
+import { HiUser } from 'react-icons/hi';
 import Category from '../redux/categories/Category';
 
 const Nav = () => (
   <Router>
-    <div className="main">
-      <header>
-        <nav>
-          <h1>Bookstore CMS</h1>
-          <ul>
-            <li>
-              <Link to="/Books">Books</Link>
-            </li>
-            <li>
-              <Link to="/Category">Category</Link>
-            </li>
-          </ul>
-        </nav>
-      </header>
+    <div className="navBg">
+      <nav className="relative flex flex-wrap bg-white items-center border-b border-slate-200 justify-between px-2 py-3 mb-3 nav-layout">
+        <h1
+          className="font-bold leading-relaxed inline-block mr-4 py-2 whitespace-nowrap text-blue-500 text-xl"
+          href="/"
+        >
+          Bookstore CMS
+
+        </h1>
+        <ul className="flex flex-col lg:flex-row list-none ">
+          <li className="nav-item px-3 py-2 font-mono flex items-center text-gray-400 text-lg hover:opacity-75">
+            <Link to="/Books">Books</Link>
+          </li>
+          <li className="nav-item px-3 py-2 font-mono flex items-center text-gray-400 text-lg hover:opacity-75">
+            <Link to="/Category">Category</Link>
+          </li>
+        </ul>
+        <div className="flex flex-col lg:flex-row list-none ">
+          <li className="p-3 border border-slate-300 text-right rounded-full text-blue-500 font-md"><HiUser /></li>
+        </div>
+      </nav>
+
```

Comment:
- [ ] Great job so far, your nav-bar design is slightly different than the one provided, kindly make sure spacing color and fonts are implemented correctly.

### Your Website
![Screen Shot 2022-09-24 at 3 16 15 PM](https://user-images.githubusercontent.com/59416136/192097461-e95a11c6-215a-4da5-a77a-2e88d5a165a0.png)

### Zeplin Template
![Screen Shot 2022-09-24 at 3 17 45 PM](https://user-images.githubusercontent.com/59416136/192097516-13eec8cd-bc78-4a56-b4d5-5062b9941909.png)

Reply:
@Wes-Isaac thanks for pointing that, i tried to the best of my ability to fix the issue